cosmwasm/packages/wasmi-runtime/src/wasm/runtime/contract.rs

```diff
@@ -200,9 +213,6 @@ impl WasmiApi for ContractInstance {
         let state_key_name = self
             .extract_vector(state_key_ptr_ptr as u32)
             .map_err(|err| {
-                // Assaf: I don't think this can happen unless the caller overrides the CosmWasm implementation of
-                // deps.api.canonical_address, not uses the CosmWasm std at all, writes the contract in a language
-                // other than rust, etc.
                 error!("read_db() error while trying to read state_key_name from wasm memory");
```

Comment:
why did you decide to remove these comment?

Reply:
Because they are not correct, and this behavior is already being tested in the CI.

Reply:
ah, awesome :+1: